src/_base/mutagen.yml.twig

```diff
@@ -15,9 +15,8 @@ sync:
         defaultFileMode: 0644
         defaultDirectoryMode: 0755
     ignore:
-      vcs: true
```

Comment:
Shouldn't we ignore git files?

Reply:
`vcs: true` ignores ANY .git, .svn, .hg, .cvs folder in the codebase.

This is a problem for the composer vendor folder, where things can be cloned from source instead of packagist.

Generally the protections that `vcs: true` provides aren't needed apart from the project repo's folder.

Reply:
aha 👍